painttiming.bs

```diff
@@ -62,9 +83,25 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Contentful</dfn>: An element counts as contentful when one or more of the following apply:
```

Comment:
What does this element refer to? If it's an [element](https://dom.spec.whatwg.org/#concept-element) defined in the DOM, then this should be linked but we also need to take [`CSSPseudoElement`](https://drafts.csswg.org/css-pseudo-4/#CSSPseudoElement-interface) into account since that's not an [element](https://dom.spec.whatwg.org/#concept-element).

Reply:
OK.